### PR TITLE
Added plenti.com

### PR DESCRIPTION
**Pull request by Foorack**
This pull request partially resolves issue #3069

### CODE REVIEW

File: src/chrome/content/rules/Plenti.com.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<ruleset name="Plenti.com">
+	<target host="plenti.com" />
+	<target host="www.plenti.com" />
+
+	<rule from="^http://(?:www\.)?plenti\.com/" to="https://www.plenti.com/"/>
```
Comment (J0WI) [inline src/chrome/content/rules/Plenti.com.xml]: This is not needed, the server redirects fine itself.

File: src/chrome/content/rules/Plenti.com.xml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<ruleset name="Plenti.com">
```
Comment (J0WI) [inline src/chrome/content/rules/Plenti.com.xml]: They also work:
marketplace.plenti.com
localdining.plenti.com
cdn2.plenti.com

**Comment by Foorack** [comment]
@J0WI Updated.

**Comment by J0WI** [comment]
Thanks!

**Pull request merged**